ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos-device-documentation.j2

```diff
@@ -92,6 +92,8 @@
 {% include 'documentation/monitor-connectivity.j2' %}
 {# Hardware TCAM Profile #}
 {% include 'documentation/tcam-profile.j2' %}
+{# Object Tracking #}
```

Comment:
"Monitor connectivity" and "Hardware TCAM Profile" sections are both with one `#` both here and in the templates. So in case these data models are active together with output-tracking, the output tracking would end up in the wrong section.
Please move up under "event-handler" to avoid this. Also please add two `##` to the description, to help identify this next time. Thanks